plugin.js

```diff
@@ -762,7 +762,7 @@
 								for(var i = 0; i < elements.length; i ++)
 								{
 									var element = elements[i];
-									var text = element.$.innerText.toString();
+									var text = element.$.textContent.toString();
```

Comment:
Using `textContent` as the default is fine, but I think there should be a `innerText` fallback (`if(CKEDITOR.env.ie && CKEDITOR.env.version === 8) ...`) until CKEditor drops support for IE8.